samples/openapi3/schema/petstore/avro-schema/InlineObject1.avsc

```diff
@@ -11,7 +11,7 @@
     },
     {
       "name": "file",
-      "type": ["null", "model.File"],
+      "type": ["null", ],
```

Comment:
Do we need to remove the comma after "null"?

Reply:
That's the bug. It's supposed to be "type": ["null", "model.File"] after the generation by avro-petstore.sh.

Since the result is the same with master, it's another modification that broke it between MR 3728 and today.

The petstore.yaml in question :

```yaml
'/pet/{petId}/uploadImage':
    post:
      tags:
        - pet
      summary: uploads an image
      description: ''
      operationId: uploadFile
      parameters:
        - name: petId
          in: path
          description: ID of pet to update
          required: true
          schema:
            type: integer
            format: int64
      responses:
        '200':
          description: successful operation
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/ApiResponse'
      security:
        - petstore_auth:
            - 'write:pets'
            - 'read:pets'
      requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                additionalMetadata:
                  description: Additional data to pass to server
                  type: string
                file:
                  description: file to upload
                  type: string
                  format: binary
```

And for some reasons, that one (which is pretty much the same) works : 
```yaml
 post:
      tags:
        - pet
      summary: Updates a pet in the store with form data
      description: ''
      operationId: updatePetWithForm
      parameters:
        - name: petId
          in: path
          description: ID of pet that needs to be updated
          required: true
          schema:
            type: integer
            format: int64
      responses:
        '405':
          description: Invalid input
      security:
        - petstore_auth:
            - 'write:pets'
            - 'read:pets'
      requestBody:
        content:
          application/x-www-form-urlencoded:
            schema:
              type: object
              properties:
                name:
                  description: Updated name of the pet
                  type: string
                status:
                  description: Updated status of the pet
                  type: string
```

The only difference that makes sense is the content type and the format field. 
It seems like the form-data field has seen some update and for some reason it can't default back to just the basic string type. 

I've crossed check with the jax-rs-cxf implementation and they have a specific mustache for it.

Reply:
I think I know where to fix it. Will merge this PR and file another PR to fix it.